### DISCUSSION TITLE
`sbt init` anyone?

### DISCUSSION THREAD

**Discussion by scf37** [sbt/sbt#7186] [Ideas]
I believe it is a shame sbt does not have one-liner to create new Scala project like other popular package managers. E.g. `npm init` or `cargo init`

**Comment by laughedelic** [comment]
sbt has `sbt new` command that uses Giter8 templates to initialize projects. Here is the documentation: https://www.scala-sbt.org/1.x/docs/sbt-new-and-Templates.html

For example:

```
$ sbt new scala/scala-seed.g8
....
name [hello]:

Template applied in ./hello
```

**Reply by scf37** [reply]
Well, it is unusable without documentation. `sbt --help` has no mention of this and even then if I have guessed somehow it is called `new` and not `init`:
```
$ sbt new
[info] welcome to sbt 1.8.2 (Oracle Corporation Java 19.0.1)
[info] loading global plugins from /home/asm/.sbt/1.0/plugins
[info] set current project to new (in build file:/tmp/sbt_17c34828/new/)
Template not found for: 
```

**Reply by mkurz** [reply]
Maybe it would be a good idea to display a link to either https://www.scala-sbt.org/1.x/docs/sbt-new-and-Templates.html#Giter8+support or https://github.com/foundweekends/giter8/wiki/giter8-templates?

**Reply by mzuehlke** [reply]
https://github.com/sbt/sbt/releases/tag/v1.9.0-RC1 adds text-based menu when `sbt new` is called without arguments.

**Reply by eed3si9n** [reply]
https://github.com/sbt/sbt/pull/7239 adds `init` as an alias to `new` as well
